CHANGELOG-LATEST.md

```diff
@@ -1,78 +1,3 @@
-
-### `usesStoreKit2IfAvailable` is now `storeKitVersion`
-
-When configuring the SDK, the `usesStoreKit2IfAvailable` parameter has been replaced by an optional `storeKitVersion: STOREKIT_VERSION` parameter. It defaults to letting the iOS SDK determine the most appropriate version of StoreKit at runtime. If you'd like to use a specific version of StoreKit, you may provide a value for `storeKitVersion` like so:
-
-```dart
-PurchasesConfiguration configuration = PurchasesConfiguration("myAPIKey");
-configuration.storeKitVersion = StoreKitVersion.storeKit1;
-
-Purchases.configure(configuration);
-```
-
-### Observer Mode is now PurchasesAreCompletedBy
-
-Version 7.0 of the SDK deprecates the term "Observer Mode" (and the APIs where this term was used), and replaces it with `PurchasesAreCompletedBy` (either RevenueCat or your app). When specifying that your app will complete purchases, you must provide the StoreKit version that your app is using to make purchases on iOS. If your app is only available on Android, you may provide any value since the native Android SDK ignores this value.
-
-You can enable it when configuring the SDK:
-
-```dart
-PurchasesConfiguration configuration = PurchasesConfiguration("myAPIKey");
-configuration.purchasesAreCompletedBy = PurchasesAreCompletedByMyApp(
-    storeKitVersion: StoreKitVersion.storeKit2,
-);
-
-Purchases.configure(configuration);
-```
-
-#### ⚠️ Observing Purchases Completed by Your App on macOS
-
-By default, when purchases are completed by your app using StoreKit 2 on macOS, the SDK does not detect a user's purchase until after the user foregrounds the app after the purchase has been made. If you'd like RevenueCat to immediately detect the user's purchase, call `Purchases.recordPurchase(productID)` for any new purchases, like so:
-
-```dart
-await Purchases.recordPurchase(productID);
-```
-
-#### Observing Purchases Completed by Your App with StoreKit 1
-
-If purchases are completed by your app using StoreKit 1, you will need to explicitly configure the SDK to use StoreKit 1:
-
-```dart
-PurchasesConfiguration configuration = PurchasesConfiguration("myAPIKey");
-configuration.purchasesAreCompletedBy = PurchasesAreCompletedByMyApp(
-    storeKitVersion: StoreKitVersion.storeKit1,
-);
-
-Purchases.configure(configuration);
-```
-
-Full migration guide to V7: [Flutter - V7 API Migration Guide](migrations/v7-MIGRATION.md)
-
-### Dependency Updates
-
-- Bump danger from 9.4.3 to 9.5.0 (#1128) via dependabot[bot] (@dependabot[bot])
-- Bump fastlane from 2.221.1 to 2.222.0 (#1126) via dependabot[bot] (@dependabot[bot])
-
-### Other Changes
-
-- [PurchaseTester] Refactor upsell screen to display all offerings (#1123) via Toni Rico (@tonidero)
+### Bugfixes
+* Update fastlane-plugin-revenuecat_internal (#1133) via Mark Villacampa (@MarkVillacampa)
```

Comment:
Oh this shouldn't be a hotfix but be in the "Other changes" section. It's probably a wrong label in the PR

Reply:
oops, I can change it manually